src/axom/sidre/examples/sidre_generateindex.cpp

```diff
@@ -111,6 +111,29 @@ void setup_blueprint_coords(DataStore* ds, Group* coords)
                          origv->getBuffer());
 }
 
+void setup_cartesian_coords(Group* coords, int domain_id)
+{
+  // 5x5 vertices, 4x4 elements
+  int nx = 5;
+  int ny = 5;
```

Comment:
Can we please make these static/global variables at the top of the file? 
That would remove the implicit assumptions about mesh size (5,5 / 4,4) throughout the file.